src/gpl/src/initialPlace.cpp

```diff
@@ -106,23 +142,157 @@ void InitialPlace::doBicgstabPlace() {
 
   // set ExtId for idx reference // easy recovery
   setPlaceInstExtId();
+
+  // Parameters that don't change with iteration
+  int m = pb_->placeInsts().size(); // number of rows of matrix A
```

Comment:
If things dont' change, suggst to make them `const`. This makes it easier for a reader to follow the code as it means one thing less to keep track of.

Reply:
Good suggestion! Helped me with the future coding.